src/router/shortUrl.ts

```diff
@@ -13,3 +19,11 @@ router.post("/urls", createUrlValidationRule, validateRequest, postUrl);
 router.get("/urls/:id", getUrlValidationRule, validateRequest, getOneUrl);
 
 router.get("/urls", validateRequest, getAllUrl);
+router.put(
+  "/urls/:id",
+  getUrlValidationRule,
+  createUrlValidationRule,
```

Comment:
For `PUT`, all data should be optional according to the spec. This validation is going to enforce you send data

Reply:
I have made the originalUrl optional